src/overlays/actors/ovl_Boss_Dodongo/z_boss_dodongo.c

```diff
@@ -55,7 +55,7 @@ static InitChainEntry sInitChain[] = {
 };
 
 void func_808C1190(s16* arg0, u8* arg1, s16 arg2) {
-    if (arg2[arg1] != 0) {
+    if (arg1[arg2] != 0) {
```

Comment:
what the heck
I don't really have a comment or question but how did `arg2[arg1]` work or match

Reply:
For some reason `index[array]` is completely allowed and defined in the C standard... And it doesn't usually affect codegen afaik (I don't think we needed it anywhere at least)